docs/google/floquet.ipynb

```diff
@@ -0,0 +1,1341 @@
+{
```

Comment:
<p>the variable <code>run_args</code>  is very unfortunate </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/3765/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='87'/>

Reply:
<p>How do you mean? I think its unfortunate that <code>PhasedFSimEngineSimulator.run</code>  and <code>Engine.run</code>  have different signatures (which necessitated <code>run_args</code>).  </p>

Reply:
The `PhasedFSimEngineSimulator.run` comes from the `cirq.Sampler` interface, not the `Engine` interface. This is aligned with another comment from Matthew that we should most likely use `cirq.Sampler` instead of `Engine`.

For the case of real engine, you would need to do something like:
```
sampler = cg.QuantumEngineSampler(engine, processor_id, cg.SQRT_ISWAP_GATESET)
```

For the case of `PhasedFSimEngineSimulator` sampler is just instance of that object. If you do that, `run_args` will disappear.

Reply:
I now use a sampler in favor of `run_args`, I think this is better, but note `engine` still needs to stick around for `cg.run_calibrations` and other calibration functions (cf. #3788). In the case of no hardware access, sampler = engine, which is a little weird.